src/fetchSites.js

```diff
@@ -9,7 +7,7 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url))
 const knownInstancesPath = path.join(__dirname, './knownInstances.json')
 const knownInstances = JSON.parse(fs.readFileSync(knownInstancesPath, 'utf-8'))
 
-const API_ENDPOINT = 'https://api.distributed.press/v1/sites'
+const API_ENDPOINT = `${knownInstances}/sites`
```

Comment:
Known instances is a list, when fetching we should iterate the list and concat the results. This currently turns the list into a string which will break when we have more than one instance.